ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -92,7 +89,6 @@ object BigSampler extends Command {
         |  --input=<path>                    Input file path or BigQuery table
         |  --output=<path>                   Output file path or BigQuery table
         |  [--fields=<field1,field2,...>]    An optional list of fields to include in hashing for sampling cohort selection
-        |  [--seed=<seed]                    An optional seed using in hashing for sampling cohort selection
```

Comment:
Thinking a bit more about our discussion earlier I am backtracking on what I said, I think it's actually best to leave this and the original hash function in for backwards compatibility. We can default to Farm hasher and allow specifying a param to use the original hasher. Only question is how to handle the seed with farm hash, potentially it can be appended to the fields we are hashing over.

Reply:
I think we should ignore it rather than use it as salt (as you're describing) -- unless you think it's important to be able to have different output with different seeds? I'm not as sure of the use case as you are...

Reply:
Yea it allows for it. If no seed is specified it will be vanilla farm hash. We can change later if it's confusing, but I think this is more clear than a param which does nothing or having the CLI have nested params

Reply:
I addressed your other comments but I think I misunderstood what you meant by this one. Let me know if you want to talk more about it!

Reply:
will do, thanks!